app/src/main/java/com/apps/adrcotfas/goodtime/Preferences.java

```diff
@@ -75,4 +89,28 @@ public String getNotificationSound() {
     public void setDisableSoundAndVibration(boolean disableSoundAndVibration) {
         mPref.edit().putBoolean(DISABLE_SOUND_AND_VIBRATION, disableSoundAndVibration).apply();
     }
+
+    public void migrateFromOldPreferences(SharedPreferences oldPref) {
+        if(oldPref.getBoolean(SESSION_DURATION, true)) {
```

Comment:
No, just basic stupidity ;)

Previously I checked FIRST_RUN, but that is not saved in the same preferences, so I needed to check another. And unfortunately, I never changed the type...

See #46 for fixing this